Makefile

```diff
@@ -64,6 +64,7 @@ license::
 	--ignore github.com/cronokirby/saferith \
 	--ignore github.com/modern-go/reflect2 \
 	--ignore github.com/prometheus/client_golang \
+	--ignore github.com/tess1o/go-ecoflow \
```

Comment:
I asked the maintainer and they meanwhile added an MIT License. So this can be removed again

Reply:
Then please do!